app/Livewire/SharedVariables/Project/Show.php

```
@@ -46,6 +52,119 @@ public function mount()
+                }
+            } else {
+                $this->project->environment_variables()->create([
+                    'key' => $key,
+                    'value' => $value,
+                    'is_multiline' => false,
+                    'is_literal' => false,
+                    'type' => 'project',
+                    'team_id' => currentTeam()->id,
+                ]);
+                $count++;
+            }
```

PHPMD (2.15.0)

The method updateOrCreateVariables uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)